### DISCUSSION TITLE
ERROR installing via docker

### DISCUSSION THREAD

**Discussion by halbius** [gramps-project/gramps-web#382] [General]
Hi,

I tried to install gramps-web on an own webserver.
When running "sudo docker-compose up -d" I get stuck with the following error:

ERROR: Get "https://ghcr.io/v2/": dial tcp 140.82.121.34:443: connect: network is unreachable

Has anyone an idea how to solve this? I could not find a solution...

I am using a [VS2-free EUserv root-server](https://www.euserv.com/en/virtual-private-server/root-vserver/v2/vs2-free.php) 
May that cause the problem? I know it's quite limited.

Thanks a lot for any help!

**Comment by DavidMStraub** [comment]
No, I suggest to contact the hoster's support.

**Reply by DavidMStraub** [reply]
To add to that, I suggest to look at the output of the much simpler command `docker pull ghcr.io/gramps-project/grampsweb` - the support might be confused by the complexity of the compose file.

**Comment by vs2-free-users** [comment]
Hi,

the basic vs2-free is IPv6 only. If you need to connect to a IPv4 Adress. You must order a IPv4 and setup it on your server.

**Reply by halbius** [reply]
Ah, I see, of course.

I found a workaround by using a nat64 nameserver.

Works fine, but it took me to the next problem: Installing things via docker seems to need lots of diskspace.
On a clean and fresh install, the installation exceeds the 10GB of the free account.

Thanks a lot for your help!
